### DISCUSSION TITLE
The menu operation always opens a new tab, which is not a good experience

### DISCUSSION THREAD

**Discussion by zhangguolin333** [WolfgangFahl/nicepdf#7] [General]
![企业微信截图_17267129778960](https://github.com/user-attachments/assets/2ff4f4da-fed5-4d0f-82f8-bdc6360fe96a)

Every time a menu item is operated, it opens a new tab instead of displaying in the content area, which is a very poor operational experience

**Comment by WolfgangFahl** [comment]
@zhangguolin333  thanks for letting us know your opion/needs - this is currently indeed the default behavior. If you want to have it changed we would need to add a settings entry or make it part of the configuration options. Please file a pull request at https://ngdemo.bitplan.com/ to get this done. If you need more hints which part of the code need to be changed feel free to ask here.
